HW1/P5/P5.txt

```diff
@@ -0,0 +1,23 @@
+# Your discussion here
+
+I made an algorithm that starts from both the root node as well as the end node, and it looks for an intersection in the
+iteration results. In this way I try to limit the number of nodes that need to be visited to find the path(s).
+
```

Comment:
Great idea! Did you do any actual benchmark to see how much faster it can be compared to search in one direction?